artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -164,6 +164,8 @@ public class NettyConnector extends AbstractConnector {
    public static final String SEC_ACTIVEMQ_REMOTING_ACCEPT = "Sec-ActiveMQRemoting-Accept";
    public static final String ACTIVEMQ_REMOTING = "activemq-remoting";
 
+   public static final String NETTY_HTTP_HEADER_PREFIX = "netty.http.header.";
```

Comment:
I think this would be better if it was relocated to `org.apache.activemq.artemis.core.remoting.impl.netty.TransportConstants` and renamed to `nettyHttpHeader.`